style.css

```diff
@@ -1 +1,299 @@
 /*  Add your styling here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Playfair Display", serif;
+}
+/* Navigation */
+
+.navbar {
+  display: none;
+  list-style: none;
+  background-color: rgb(247, 247, 247);
+  margin: 12px 115px 5px 45px;
+  padding: 8px 7px 8px 900px;
+}
+
+.nav-item {
+  padding: 0.5rem 0.3rem;
+}
+
+.nav-link {
+  color: rgb(160, 155, 155);
+  text-decoration: none;
+  font-size: 0.7rem;
+  background-color: rgb(219, 217, 217);
+  padding: 4px 22px;
+  color: rgb(7, 7, 7);
+  transition: background-color 2s;
+}
+.nav-link:hover {
+  color: #f5f0f0;
+  cursor: pointer;
+  background-color: rgb(48, 48, 48);
+}
+/* grid */
+.content {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "products"
+    "footer";
+}
+header {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "logo logo logo logo logo ham"
+    ". . par par par par"
+    ". . par1 par1 par1 par1";
```

Comment:
Be careful not to use too many grids where flex would be suitable, flex can be simpler which means your code is easier to read and maintain, especially when you're writing code in a team